src/metabase/driver/postgres.clj

```diff
@@ -163,7 +163,13 @@
     driver.common/advanced-options-start
     (assoc driver.common/additional-options
            :placeholder "prepareThreshold=0")
-    driver.common/default-advanced-options]
+    driver.common/default-advanced-options
+    {:name         "database-enable-actions"
+     :type         :boolean
+     :default      false
+     :display-name (deferred-tru "Enable experimental actions")
+     :description  (deferred-tru "Whether to enable using the new experimental Actions features for this database. (Actions must also be enabled globally.)")
+     :visible-if   {"settings.experimental-enable-actions" true "advanced-options" true}}]
```

Comment:
@camsaul is there a correct way to add a dependency on global settings on the backend or this is a frontend only value?

Reply:
Since this is a Database-local Setting and not something that goes in the details map you should probably just put all this info in the FE somewhere